examples/simple-usage/recipe-type.ts

```diff
@@ -1,34 +1,34 @@
-import { Field, ObjectType, Int, Float } from "type-graphql";
+import { Field, ObjectType } from "type-graphql";
+import { GraphQLInt, GraphQLFloat } from "graphql";
 
 @ObjectType({ description: "Object representing cooking recipe" })
 export class Recipe {
   @Field()
   title: string;
 
-  @Field(type => String, { nullable: true, deprecationReason: "Use `description` field instead" })
+  @Field(() => String, { nullable: true, deprecationReason: "Use 'description' field instead" })
   get specification(): string | undefined {
     return this.description;
   }
 
   @Field({ nullable: true, description: "The recipe description with preparation info" })
   description?: string;
 
-  @Field(type => [Int])
+  @Field(() => [GraphQLInt])
   ratings: number[];
 
   @Field()
   creationDate: Date;
 
-  @Field(type => Int)
+  @Field(() => GraphQLInt)
```

Comment:
Why removing the scalar alias?

Reply:
It is good for you to prepend `returns` with `_returns` so ESLint does not complain and also we retain the retunr keyword?

Reply:
Then we would also start using `GraphQLString` and `GraphQLBoolean` instead of `String` and `Boolean` JS constructors.

Reply:
Yup/. Add in TODO i nthe issue?

Reply:
> All libraries and examples (outside TypeGrapQL) use the default from graphql.

I don't like to compare to others. All the blog posts about `TypeGraphQL` and production code I've been working with in several companies uses the aliases as showed in the docs and examples. Even my `typegraphql-prisma` library 😛 

So I don't like that idea. It's a big unnecessary breaking change. 
I think All we should do is to show that `GraphQLXYZ` scalars approach as an alternative one.

Reply:
Okok

Reply:
Fixed. Moreover have you any idea how to move examples dependencies to the respective directory? The change I made is broken but maybe using Lerna or something similar?